templates/zsh.txt

```diff
@@ -59,7 +59,9 @@ function __zoxide_z() {
             echo "zoxide: \\$OLDPWD is not set"
             return 1
         fi
-    elif ! { [ "$#" -eq 1 ] && __zoxide_cd "$1" &>{{ Opts::DEVNULL }}; }; then
+    elif [ "$#" -eq 1 ] && (cd "$1" &>{{ Opts::DEVNULL }}) then
```

Comment:
The cd hook here calls zoxide upon completion, so `zoxide add` ends up getting called twice.